webrtc.html

```diff
@@ -1262,30 +1265,42 @@
             <a href="#widl-RTCIceCandidate-sdpMid">sdpMid</a></code> and <code>
             <a href=
             "#widl-RTCIceCandidate-sdpMLineIndex">sdpMLineIndex</a></code>; the
-            rest are ignored.</p>
+            rest are ignored. When the method is invoked, the User Agent MUST
+            run the following steps:</p>
 
             <ol>
               <li>
-                <p>Let <var>connection</var> be the <code>
-                <a>RTCPeerConnection</a></code> object on which the method was
-                invoked.</p>
+                <p>Let <var>candidate</var> be the methods argument.</p>
               </li>
 
               <li>
-                <p>If <var>connection</var>'s [[<a>isClosed</a>]] slot is
-                <code>true</code>, return a promise rejected with an
-                <code>InvalidStateError</code>.</p>
+                <p>If <var>candidate</var> is not <code>null</code> but is
+                missing values for both <var>sdpMid</var> and
+                <var>sdpMLineIndex</var>, return a promise rejected with a
+                <code>TypeError</code>.</p>
               </li>
 
               <li>
-                <p>Let <var>candidate</var> be the methods argument.</p>
+                <p>Return the result of <a href=
+                "#enqueue-an-operation">queuing</a> <a>addIceCandidate's queued
+                steps</a>.</p>
               </li>
+            </ol>
+
+            <p>The following steps defines <dfn>addIceCandidate's queued
+            steps</dfn>:</p>
 
+            <ol>
               <li>
-                <p>If <var>candidate</var> is not <code>null</code> but is
-                missing values for both <var>sdpMid</var> and
-                <var>sdpMLineIndex</var>, return a promise rejected with a
-                <code>TypeError</code>.</p>
+                <p>Let <var>connection</var> be the <code>
+                <a>RTCPeerConnection</a></code> object on which the method was
+                invoked.</p>
```

Comment:
I think it would make sense to move this step out of the enqueued steps, to the steps above, to be next to `candidate` and the others. Otherwise the context for this text seems wrong.